source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetTemplateCommand.cs

```diff
@@ -0,0 +1,238 @@
+﻿using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.Evaluations
+{
+    public class GetTemplateCommand : Command<Template>
```

Comment:
Maybe `GetEvaluationTemplateCommand` suits as a better name? (But it's still under the `EvalForm` namespace, so it may be debatable.)